compiler/testData/integration/ant/jvm/forkOnError/build.xml

```diff
@@ -0,0 +1,19 @@
+<project name="Ant Task Test" default="build">
+    <taskdef resource="org/jetbrains/kotlin/ant/antlib.xml" classpath="${kotlin.lib}/kotlin-ant.jar"/>
+    <target name="build">
+        <kotlinc src="${test.data}" output="${temp}/fork.jar" fork="true">
+            <jvmarg value="-Xms64m"/>
+            <jvmarg value="-Xmx128m"/>
+            <jvmarg value="-Xmn32m"/>
+            <jvmarg value="-XX:ReservedCodeCacheSize=64m"/>
+            <jvmarg value="-XX:CICompilerCount=4"/>
+            <jvmarg value="-XX:+PrintCommandLineFlags"/>
+
+            <compilerarg line="-jvm-target 1.8"/>
```

Comment:
Looks like both in this and in the other test, we're not checking that the compiler arguments are properly applied.

I propose to do the following to improve that:
- in this test, use just one compiler argument `-Werror`, and change the error in the code to some warning. For example, replace `println1(...)` with `println(... as String)`.
- in the other test (`integration/ant/jvm/fork/`), change the code in the same way but use just one compiler argument `-nowarn`.

This will test not only the non-zero exit code and the error message text/paths, but also the fact that compiler arguments are applied.

Reply:
Thanks for your suggestion! Now the tests look better.